src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -743,6 +743,34 @@ private void ProcessCachedGroupOnWide(WideViewHeaderInfo wvhi, List<PacketInfoDa
             _formattingHint = hint;
         }
 
+        static private int _failedToReadConsoleWidth = 0;
+
+        static private int GetConsoleWindowWidth(int columnNumber)
+        {
+            if (columnNumber == int.MaxValue)
+            {
+                // Tables and Wides need to use spaces for padding to maintain table look even if console window is resized.
+                // For all other output, we use int.MaxValue if the user didn't explicitly specify a width.
+                // If we detect that int.MaxValue is used, first we try to get the current console window width.
+                // However, if we can't read that (for example, implicit remoting has no console window), we default
+                // to something reasonable: 120 columns.
```

Comment:
I'd move the comment before the method and add clarification about `_failedToReadConsoleWidth` and remote session.

Reply:
OK.

However, I am not good at English and I do not have much confidence to make appropriate comments.
Is there any good comment plan?

Reply:
@SteveL-MSFT Could you please help with the comment?

Reply:
@stknohg Your English is actually quite good.  I believe @iSazonov is asking to move the comment to where line 748 right before the method.  You should probably just add some text before line 746 like:

```powershell
// In cases like implicit remoting, there is no console so reading the console width results in an exception.
// We cache this value rather than handling the exception (on the target) everytime as a performance improvement.
```

Reply:
@SteveL-MSFT Thank you.I will add your comment.

Reply:
@stknohg I think this PR is almost there.  Just need to move the comment from lines 756-760 to right before line 750 since the comment applies to the entire method and not just the `if` statement within the method